### DISCUSSION TITLE
Using RIFM to mask input and rhf

### DISCUSSION THREAD

**Discussion by hubertokf** [react-hook-form/react-hook-form#4094] [Q&A]
Has someone used react-hook-form with [RIFM](https://github.com/realadvisor/rifm) lib?

RIFM is a mask lib.

I'm having dificulties using react-hook-form + material-ui + react-text-mask. RIFM seems to be a simpler way to mask inputs, no need to deal with refs.

**Comment by bluebill1049** [comment]
I think you can build a component with RIFM, and wrapped with controller. there is a similar mask input example here: https://codesandbox.io/s/react-hook-form-v6-controller-qsd8r
